### PR TITLE
Add reducers and actions.

### PR DESCRIPTION
**Pull request by OyePriscilla**
## Changes made in this PR:

- Defined action types for adding, checking, and removing a book.
- Defined state changes for the actions created.
- Reducer exported as default.
- Configured the Redux Store.
- Combined both reducers into a root reducer by using the configureStore function.
- Ducks pattern was observed for the file arrangement.
- App deployed with netlify.

### CODE REVIEW

File: src/redux/categories/categories.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Actions
+const CHECK = 'bookstore/books/CHECK';
+
+// Reducer
+export default function categoriesReducer(state = [], action = {}) {
+  switch (action.type) {
+    case CHECK:
+      return 'Under Construction';
+    default: return state;
```
Comment (dasileker) [inline src/redux/categories/categories.js:9]: - [ ]Please make sure that this **action.type** for `under Construction` is rendered in the  category page not **check back.**

Capture web_16-8-2022_14239_localhost

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @OyePriscilla,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- Linters are green 👌.
- PR is very detailed  👌.
- Commit are consistent 👌.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/categories/categories.js (hunk 1 above)
Comment (OyePriscilla) [reply src/redux/categories/categories.js:9]: Ok, noted.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @OyePriscilla, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OyePriscilla** [comment]
Thanks @ShadyShawkat .

**Pull request merged**